### PR TITLE
Fix command to install weavenet plugin

### PR DESCRIPTION
**Pull request by nirvanagit**
Fix command to install weave net network plugin

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
/lgtm 
I checked the command on https://www.weave.works/docs/net/latest/kubernetes/kube-addon/#pod-network " Changing Configuration Options" section.

**Comment by nirvanagit** [comment]
Can this be merged?

File: content/en/docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -325,9 +325,7 @@ Weave Net sets hairpin mode by default. This allows Pods to access themselves vi
 if they don't know their PodIP.
 
 ```shell
-export kubever=$(kubectl version | base64 | tr -d '
-')
-kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version="
+kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '')"
```
Comment (luxas) [inline content/en/docs/setup/independent/create-cluster-kubeadm.md]: I think it should be `kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"` (note the \n)

**Comment by mistyhacks** [comment]
Please address @luxas comment and resolve the conflicted file. Thanks!

**Comment by zacharysarah** [comment]
@nirvanagit Any updates?

**Comment by mistyhacks** [comment]
I'll carry this.

**Comment by mistyhacks** [comment]
I rebased and integrated the feedback.

/lgtm
/approve

**Comment by nirvanagit** [comment]
Thanks @mistyhacks

**Pull request merged**